www/src/components/layout.js

```diff
@@ -151,6 +152,7 @@ class DefaultLayout extends React.Component {
           <meta name="twitter:site" content="@gatsbyjs" />
           <meta name="og:type" content="website" />
           <meta name="og:site_name" content="GatsbyJS" />
+          <meta name="og:image" content={`https://gatsbyjs.org${gatsbyIcon}`} />
```

Comment:
Please leave off the gatsbyjs.org so this will work on test deploys or locally (also fwiw, it's www.gatsbyjs.org).

Reply:
It originally was relative path, but I think that path need to be absolute - didn't test it just searched for issues on the web and found few reports that relative paths don't work (granted it was from 2 years ago)